.github/workflows/release.yml

```diff
@@ -55,10 +56,10 @@ jobs:
         uses: changesets/action@v1
         with:
           # Put date in the title and commit message
-          title: "Theme Tools Release — ${{ steps.date.outputs.date }}"
-          commit: "Theme Rools Release — ${{ steps.date.outputs.date }}"
+          title: "(Demo) Theme Tools Release — ${{ steps.date.outputs.date }}"
+          commit: "(Demo) Theme Rools Release — ${{ steps.date.outputs.date }}"
           # When there are no changesets, this gets called
-          publish: yarn changeset publish
+          publish: echo yarn changeset publish
```

Comment:
For demo purposes, a publish to npm is mocked with `echo yarn changeset publish`